python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -467,15 +394,20 @@ class OpRuntimeMetrics(metaclass=OpRuntimesMetricsMeta):
         description="Time spent in task output backpressure.",
         metrics_group=MetricsGroup.TASKS,
     )
-    task_completion_time: list[int] = metric_field(
-        default_factory=list,
-        description="Time spent running tasks to completion.",
+    task_completion_time_total: float = metric_field(
```

Comment:
@alexeykudinkin , is it okay if we rename this to `task_completion_time_total`? I think this name is slightly better than calling the other metric `task_completion_time_histogram`